src/common/cockpitchannel.c

```diff
@@ -242,6 +335,35 @@ cockpit_channel_actual_send (CockpitChannel *self,
 
   cockpit_transport_send (self->priv->transport, self->priv->id, payload);
 
+  /* A wraparound of our gint64 size */
```

Comment:
surprised i didn't notice this the first time through.  really? :)

Reply:
Would you prefer a g_assert()? What would be better here?

Reply:
assert seems reasonable, but could also just leave it now.  it's definitely dead code, though.